contrib/babelfishpg_tds/src/backend/fault_injection/fault_injection_tests.c

```diff
@@ -264,5 +279,6 @@ TEST_LIST = {
 	{"parsing_tamper_rpc_parameter_datatype", ParseRpcType, 0, &parsing_tamper_rpc_parameter_datatype},
 	{"pre_parsing_throw_error", PreParsingType, 0, &throw_error},
 	{"post_parsing_throw_error", PostParsingType, 0, &throw_error},
+	{"buffer_overflow_test",PreParsingType,0, &throw_error_buffer},
```

Comment:
This just declares the test. You need to invoke the test from the JDBC test framework - https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/test/JDBC/input/babel_tds_fault_injection.sql

Reply:
ok , will add it